packages/autorest.python/autorest/codegen/models/operation.py

```diff
@@ -82,6 +82,9 @@ def __init__(
             "isLroInitialOperation", False
         )
         self.include_documentation: bool = not self.is_lro_initial_operation
+        self.is_public: bool = self.yaml_data.get("isPublic", True)
```

Comment:
can we switch the variable to `internal`? That way we keep it the same as typespec